### PR TITLE
Update download-and-copy-artifacts-using-the-command-step.md

### PR DESCRIPTION
**Pull request by thisrohangupta**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by roshnisarangadharan — DISMISSED** [review]

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md [hunk 1]
```diff
@@ -341,6 +341,151 @@ The WinRM deployment type supports Download Artifact only. You cannot use Cope A
...
+     smtp_sasl_auth_enable = yes
+     smtp_sasl_password_maps = hash:/etc/postfix/sasl_passwd
+     smtp_sasl_security_options = 
+     smtp_tls_CAfile = /etc/ssl/certs/ca-certificates.crt
+     smtp_use_tls = yes
+     EOT
+     sudo chmod 600 /etc/postfix/sasl_passwd
+     sudo postmap /etc/postfix/sasl_passwd
+     sudo systemctl restart postfix.service
+     echo "This is the body of an encrypted email" | mail -A hello.jar -s "log file"  rohan@harness.io
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md:394]:
```suggestion
     echo "This is the body of an encrypted email" | mail -A hello.jar -s "log file"  johndoe@xyz.io
```

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md [hunk 2]
```diff
@@ -341,6 +341,151 @@ The WinRM deployment type supports Download Artifact only. You cannot use Cope A
...
+   * Set the **Command Type** to **Script**.
+   * Enter the **Working Directory** path where the attachment resides. The path can be a fixed value, runtime input, or an expression.
+   * In **Select script location**, select **Inline**.
+   * In **Script Type**, select **Bash** and enter the following command: 
+     
+     :::info
+
+     * The `relayhost` host configuration in the following command can be any SMTP port and server.
+     
+     * You can define the body of the message with an echo statement like this, `echo "This is the body of an encrypted email" | mail -A hello.jar -s "log file"  rohan@harness.io`. You can provide harness expressions to resolve the body of the message. The email address can be provided as a user's email or a Distribution List email.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md:370]:
```suggestion
     * You can define the body of the message with an echo statement like this, `echo "This is the body of an encrypted email" | mail -A hello.jar -s "log file"  johndoe@xyz.io`. You can provide harness expressions to resolve the body of the message. The email address can be provided as a user's email or a Distribution List email. 
```

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md [hunk 3]
```diff
@@ -341,6 +341,151 @@ The WinRM deployment type supports Download Artifact only. You cannot use Cope A
...
+                                smtp_sasl_auth_enable = yes
+                                smtp_sasl_password_maps = hash:/etc/postfix/sasl_passwd
+                                smtp_sasl_security_options = 
+                                smtp_tls_CAfile = /etc/ssl/certs/ca-certificates.crt
+                                smtp_use_tls = yes
+                                EOT
+                                sudo chmod 600 /etc/postfix/sasl_passwd
+                                sudo postmap /etc/postfix/sasl_passwd
+                                sudo systemctl restart postfix.service
+                                echo "This is the body of an encrypted email" | mail -A hello.jar -s "log file"  srgupta5328@gmail.com
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md:459]:
```suggestion
                                echo "This is the body of an encrypted email" | mail -A hello.jar -s "log file"  johndoe@xyz.io
```

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md [hunk 4]
```diff
@@ -341,6 +341,151 @@ The WinRM deployment type supports Download Artifact only. You cannot use Cope A
...
 
+```BASH
+$ sudo apt-get install postfix mailutils libsasl2-2 ca-certificates libsasl2-modules
+```
+
+#### Configuration
+
+1. Create a deployment stage with an SSH type service and infrastructure definition.
+2. Add a Command step under the **Executions** tab.
+3. In **Step Parameters** enter a name and timeout duration.
```
Comment (aimurphy) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md:358]:
```suggestion
3. In **Step Parameters**, enter a name and timeout duration. 
```

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md [hunk 5]
```diff
@@ -341,6 +341,151 @@ The WinRM deployment type supports Download Artifact only. You cannot use Cope A
...
+2. Add a Command step under the **Executions** tab.
+3. In **Step Parameters** enter a name and timeout duration. 
+4. In **Run the following commands**, select **Add Command**.
+   * Enter a command name.
+   * Set the **Command Type** to **Script**.
+   * Enter the **Working Directory** path where the attachment resides. The path can be a fixed value, runtime input, or an expression.
+   * In **Select script location**, select **Inline**.
+   * In **Script Type**, select **Bash** and enter the following command: 
+     
+     :::info
```
Comment (aimurphy) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md:366]: I don't think the info box is necessary here. This isn't extra information; this is information about how to complete this step. If it is actually extra/non-essential info, I would put it after the command block.

If you are worried about too many bullets, I would promote some of the bullets to steps, like:

```
4. In **Run the following commands**, select **Add Command**, enter a command name, and set the **Command Type** to **Script**.
5. Enter the **Working Directory** path...
```

**Review by aimurphy — APPROVED** [review]
Approved with some minimalism/info architecture type suggestions

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md [hunk 6]
```diff
@@ -341,6 +341,139 @@ The WinRM deployment type supports Download Artifact only. You cannot use Cope A
 | Azure Artifacts | Maven 2.0, NuGet | Y | Y |
 | Custom Repository | All | N/A | N (use the Exec script to use the metadata to copy artifact to target host) |
 
+## Use case: Sending an email from a server with an attachment
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md:344]:
```suggestion
## Use case: Sen emails from a server with an attachment
```

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md [hunk 7]
```diff
@@ -341,6 +341,139 @@ The WinRM deployment type supports Download Artifact only. You cannot use Cope A
 | Azure Artifacts | Maven 2.0, NuGet | Y | Y |
 | Custom Repository | All | N/A | N (use the Exec script to use the metadata to copy artifact to target host) |
 
+## Use case: Sen emails from a server with an attachment
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/download-and-copy-artifacts-using-the-command-step.md:344]:
```suggestion
## Use case: Send emails from a server with an attachment
```

**Review by roshnisarangadharan — APPROVED** [review]

**Comment by roshnisarangadharan** [comment]
@thisrohangupta let me know when to merge this.

**Pull request merged**